leases/services/invoice/berth.py

```diff
@@ -0,0 +1,36 @@
+from datetime import date
+
+from django.db.models import QuerySet
+
+from payments.enums import OrderStatus
+from payments.models import BerthProduct, Order
+
+from ...models import BerthLease
+from ...utils import calculate_season_end_date, calculate_season_start_date
+from .base import BaseInvoicingService
+
+
+class BerthInvoicingService(BaseInvoicingService):
```

Comment:
... and no changes to BerthInvoicingService (besides moving it to a different file) :-)